web-ui/src/pages/MemberProfilePage.jsx

```diff
@@ -15,6 +15,7 @@ import { getGuildsForMember } from '../api/guild';
 import { getAvatarURL } from '../api/api.js';
```

Comment:
This page is accessed by selecting DIRECTORY ... People and clicking on a card for a specific team member.